shuup/discounts/management/commands/sync_catalog_campaigns.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shuup.
+#
+# Copyright (c) 2012-2018, Shuup Inc. All rights reserved.
+#
+# This source code is licensed under the OSL-3.0 license found in the
+# LICENSE file in the root directory of this source tree.
+import datetime
+import itertools
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.utils.translation import activate
+
+from shuup.admin.forms.fields import WeekdayField
+from shuup.campaigns.models import CatalogCampaign
+from shuup.campaigns.models.catalog_filters import (
+    CategoryFilter, ProductFilter
+)
+from shuup.campaigns.models.context_conditions import (
+    ContactCondition, ContactGroupCondition, HourCondition
+)
+from shuup.campaigns.models.product_effects import (
+    ProductDiscountAmount, ProductDiscountPercentage
+)
+from shuup.core.models import (
+    Category, CompanyContact, ContactGroup, PersonContact, Product
+)
+from shuup.discounts.models import Discount, HappyHour, TimeRange
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
```

Comment:
Can't be migrations since related objects are too hard. Migrating this is up to who ever does the update. There is always option to keep using the old catalog campaigns too until those are "consumed".

Reply:
well, you removed it from admin, so maybe you can put it back and deprecated in some way, like preventing to create new ones?